src/components/Popups/PopupCursor.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+import messages from './messages';
+import PopupBase from './PopupBase';
+import { Options } from './Popper';
+import './PopupCursor.scss';
+
+export type Props = {
+    isActive: boolean;
+};
+
+const options: Partial<Options> = {
+    modifiers: [
+        {
+            name: 'offset',
+            options: {
+                offset: [0, 20],
+            },
+        },
+    ],
+    placement: 'right',
+};
+
+export default class PopupCursor extends React.PureComponent<Props> {
+    popupRef = React.createRef<PopupBase>();
+
+    virtualElement = {
+        getBoundingClientRect: () => new DOMRect(0, 0, 0, 0),
+    };
+
+    constructor(props: Props) {
+        super(props);
+        document.addEventListener('mousemove', this.updatePosition);
+    }
+
+    componentWillUnmount(): void {
+        document.removeEventListener('mousemove', this.updatePosition);
+    }
+
+    updatePosition = ({ clientX: x, clientY: y }: { clientX: number; clientY: number }): void => {
+        const { isActive } = this.props;
```

Comment:
Is this something this component should be handling or should the parent simply not render it?